tsdb/functions.go

```diff
@@ -432,9 +435,9 @@ func MapMean(input *MapInput) interface{} {
 		out.Count++
 		switch v := item.Value.(type) {
 		case float64:
-			out.Mean += (v - out.Mean) / float64(out.Count)
+			out.Total += v
```

Comment:
What is happening here? It looks like you're changing how this works, is that correct? If so, why?